javatests/arcs/showcase/instant/EventsToday.kt

```diff
@@ -0,0 +1,31 @@
+@file:Suppress("EXPERIMENTAL_FEATURE_WARNING", "EXPERIMENTAL_API_USAGE")
+
+package arcs.showcase.instant
+
+import arcs.jvm.host.TargetHost
+import arcs.sdk.ArcsDuration
+import arcs.sdk.ArcsInstant
+import arcs.showcase.ShowcaseHost
+
+typealias Agenda = AbstractEventsToday.EventsToday_Agenda
+
+@TargetHost(ShowcaseHost::class)
+class EventsToday : AbstractEventsToday() {
+
+    override fun onReady() {
+        handles.events.fetchAll().forEach {
+            if (
+                (it.start <= ArcsInstant.now().plus(ArcsDuration.ofDays(1))) &&
+                (ArcsInstant.now() <= it.end)
+            ) {
+                handles.agenda.store(
```

Comment:
More Kotlin-y would be:

`handles.agenda.storeAll(handles.events.fetchAll().filter {...}.map {...}`